### PR TITLE
Feature/mysql query insights

### PR DESCRIPTION
**Pull request by nurairasia**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -95,9 +97,12 @@ module "google_mysql_db" {
...
     for array_index in range(var.read_replica_count) : {
-      name = array_index
-      tier = var.instance_size_read_replica
-      zone = local.zone_read_replica
+      name                  = array_index
+      tier                  = var.instance_size_read_replica
+      zone                  = local.zone_read_replica
+      availability_type     = null
+      disk_autoresize       = true
+      disk_autoresize_limit = 0
```
Comment (syedrakib) [inline main.tf]: These values should come from variables (like how `var.instance_size_read_replica` was used)

File: main.tf [hunk 2]
```diff
@@ -95,20 +97,23 @@ module "google_mysql_db" {
...
     for array_index in range(var.read_replica_count) : {
-      name = array_index
-      tier = var.instance_size_read_replica
-      zone = local.zone_read_replica
+      name                  = array_index
+      tier                  = var.instance_size_read_replica
+      zone                  = local.zone_read_replica
+      availability_type     = null
+      disk_type             = "PD_SSD"
+      disk_autoresize       = true
```
Comment (syedrakib) [inline main.tf]: Why removing `var.disk_auto_resize_read_replica` ?

Comment (nurairasia) [reply main.tf]: added it back

File: main.tf (hunk 1 above)
Comment (nurairasia) [reply main.tf]: disk_autoresize_limit is set to 0 in master (unlimited) so we should keep it this way in read replica also

**Comment by nurairasia** [comment]
similar PR - https://github.com/airasia/terraform-google-mysql_db/pull/10

**Pull request closed without merging**